thrift/sampling.thrift

```diff
@@ -20,6 +20,7 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
+namespace cpp jaegertracing.thrift.sampling_manager
```

Comment:
inconsistent with agent, it had thrift as a suffix

Reply:
OK sure